recipes/akt/build.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+sed -i 's/\-mpopcnt//g' Makefile
```

Comment:
This should be wrapped in `if not x86_64; then` ...

Reply:
ok, I will fix it